chart/README.md

```diff
@@ -25,7 +25,11 @@ A Helm chart for deploying Prometheus configured to use TimescaleDB as compresse
 The recommended way to deploy tobs is through the [CLI tool](/cli). However, we also
 support helm deployments that do not use the CLI.
 
-The following command will install Prometheus, TimescaleDB, Promscale Connector, and Grafana
+## Creating Secrets
+
+By default, timescaledb helm chart doesn't create its own secrets. So please follow [these instructions][timescaledb-secrets] to create timescaledb-secrets. If you use tobs CLI this is taken care for you.
+
+The following command will install Kube-Prometheus, TimescaleDB, and Promscale Connector
```

Comment:
It doesn't make sense for this line to be under the "Creating Secrets" header. I think this needs to be re-worked to make sense